src/Components/Main.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const Main: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
+      {/* This is a completed todo */}
+      {todos.map(todo => (
```

Comment:
It would be nice to create a separate component for Todo item

Reply:
But I have already created the component ' Main ' for Todo Item.
Maybe just rename it to ' Todos ' or ' TodosList ' ?

Reply:
Yes, TodoList is a better naming, but also you can create component for each Todo, to make it like this

```suggestion
      {todos.map(todo => <Todo todo={todo} key.. />)
```

Reply:
Roger! I'll try)